Description: Fix FTBFS on kfreebsd due to wrong configuration of iconv args
Bug: https://code.google.com/p/libtorrent/issues/detail?id=487
Bug-Debian: http://bugs.debian.org/712797
Forwarded: http://sourceforge.net/p/libtorrent/code/8527/
Origin: vendor
Author: Pino Toscano <pino@debian.org>

--- a/include/libtorrent/config.hpp
+++ b/include/libtorrent/config.hpp
@@ -183,9 +183,11 @@
 #define TORRENT_CLOSE_MAY_BLOCK 1
 #endif
 #else
+#ifndef __GLIBC__
 // FreeBSD has a reasonable iconv signature
 #define TORRENT_ICONV_ARG (const char**)
 #endif
+#endif
 #define TORRENT_HAS_FALLOCATE 0
 #define TORRENT_USE_IFADDRS 1
 #define TORRENT_USE_SYSCTL 1
